arm:
  type: ur10
sensors:
  logical_camera_1:
    type: logical_camera
    pose:
      xyz: [-0.3, 1.0, 1.45]
      rpy: [0, 'pi/2', 0]
  logical_camera_2:
    type: logical_camera
    pose:
      xyz: [0.26, 3.1, 1.49]
      rpy: [0, 'pi/2', 0]
  logical_camera_3:
    type: logical_camera
    pose:
      xyz: [0.26, -3.1, 1.49]
      rpy: [0, 'pi/2', 0]  #['pi/2', 'pi/2', 'pi']
  logical_camera_4:
    type: logical_camera
    pose:
      xyz: [-0.3, 0.24, 1.45]
      rpy: [0, 'pi/2', 0]
  logical_camera_5:
    type: logical_camera
    pose:
      xyz: [-0.3, -0.5, 1.45]
      rpy: [0, 'pi/2', 0]
  logical_camera_6:
    type: logical_camera
    pose:
      xyz: [-0.3, -1.32, 1.45]
      rpy: [0, 'pi/2', 0]
  logical_camera_7:
    type: logical_camera
    pose:
      xyz: [1.21, 2.7, 1.65]
      rpy: [0, 'pi/2', 0]
  logical_camera_8:
    type: logical_camera
    pose:
      xyz: [1.21, 1.8, 1.65]
      rpy: [0, 'pi/2', 0]
  laser_profiler_1:
    type: laser_profiler
    pose:
      xyz: [1.21, 4.75, 1.59]
      rpy: ['-pi', 'pi/2', 'pi/2']